common_features.mk

```diff
@@ -730,3 +730,8 @@ ifeq ($(strip $(USBPD_ENABLE)), yes)
         endif
     endif
 endif
+
```

Comment:
I know this is kinda trivial, but could you bump this above the joystick logic (line 693) so its with all the other `process_keycode` stuff.

Reply:
Solved in https://github.com/qmk/qmk_firmware/pull/11036/commits/ff8d642ee787184eb3ead694239a39ceccee1c88